db/migrate/20171010172724_add_docker_manifest_list.rb

```diff
@@ -19,8 +19,7 @@ def up
       t.references :docker_manifest, :null => false
       t.timestamps
     end
-
-    remove_foreign_key :katello_docker_tags, :docker_manifest
+    remove_foreign_key :katello_docker_tags, :katello_docker_manifests
```

Comment:
I don't understand how this was working before. Hasn't the table name always been `katello_docker_manifests`?

Reply:
I am not sure how it was working either, the table name has been always the same.

Reply:
@xprazak2 note, i pulled this change out so that we can go ahead and merge it to get rails 5 tests passing: https://github.com/Katello/katello/pull/7102